charts/hocs-toolbox/values.yaml

```diff
@@ -1,6 +1,14 @@
 ---
 nameOverride: hocs-toolbox
 
+selector:
+  ingress:
+    required: false
+  outbound:
+    required: false
```

Comment:
`hocs-toolbox` has been used to get S3 documents before. I presume outbound allows for S3 lookups and would need to be required...

Reply:
I'll add it, it's our tool afterall